metricbeat/docs/running-on-kubernetes.asciidoc

```diff
@@ -103,8 +112,24 @@ specify the following settings under `kubernetes.yml` in the `data` section:
       hosts: ["https://${NODE_NAME}:10250"]
       bearer_token_file: /var/run/secrets/kubernetes.io/serviceaccount/token
       ssl.certificate_authorities:
-        - /var/run/secrets/kubernetes.io/serviceaccount/service-ca.crt
+        - ${VALID_CA}
 -----
+NOTE: `VALID_CA` can be any CA that is valid so as to reach out the Kubelet API,
```

Comment:
Nit. Use a different format for this, so it doesn't look like an environment variable that should be defined somewhere.
```suggestion
        - /path/to/kubelet-service-ca.crt
-----
NOTE: `kubelet-service-ca.crt` can be any CA bundle that contains the issuer of the certificate used in the Kubeler API,
```